code/__DEFINES/say.dm

```diff
@@ -45,7 +45,10 @@
 
 #define MODE_MONKEY "monkeyhive"
 
-#define MODE_SING "%"
+#define MODE_SING "sing"
```

Comment:
this change isn't really needed, % works and is consistent with other saymods

Reply:
No its not?
#define SPAN_ROBOT "robot"
#define SPAN_YELL "yell"
#define SPAN_ITALICS "italics"
#define SPAN_SANS "sans"
#define SPAN_PAPYRUS "papyrus"
#define SPAN_REALLYBIG "reallybig"
#define SPAN_COMMAND "command_headset"
#define SPAN_MEGAPHONE "megaphone"
#define SPAN_CLOWN "clowntext"

 #define WHISPER_MODE "the type of whisper"
#define MODE_WHISPER "whisper"
#define MODE_WHISPER_CRIT "whispercrit"

Reply:
the other saymods as in ; : *

Reply:
Thats.. not what this is for...

Reply:
mode sing isn't just sign emote, it actually changes the output of your message

Reply:
https://cdn.discordapp.com/attachments/801432004058939422/917809046730469386/unknown.png

I have no idea what you're talking about